lodestar.code-workspace

```diff
@@ -0,0 +1,12 @@
+{
+  "settings": {
+    "window.title": "${activeEditorShort}${separator}${rootName}${separator}${profileName}${separator}[${activeRepositoryBranchName}]${separator}[${activeRepositoryBranchName}]",
```

Comment:
I personally like having this settings when I have multiple windows of VSCode open. Thought to share with everyone.

Reply:
can you share a screenshot how this looks like

Reply:
<img width="631" alt="image" src="https://github.com/user-attachments/assets/c208abe5-d0af-4b22-b698-d3b7e3c34b60">